docs/sources/configure/_index.md

```diff
@@ -1914,6 +1914,55 @@ hedging:
 # CLI flag: -store.index-cache-validity
 [index_cache_validity: <duration> | default = 5m]
 
+congestion_control:
+  # Use storage congestion control (default: disabled).
+  # CLI flag: -store.enabled
+  [enabled: <boolean> | default = false]
+
+  controller:
+    # Congestion control strategy to use (default: none, options: 'aimd').
+    # CLI flag: -store.congestion-control.strategy
+    [strategy: <string> | default = ""]
+
+    aimd:
+      # AIMD starting throughput window size: how many requests can be sent per
```

Comment:
I'd reference https://en.wikipedia.org/wiki/Additive_increase/multiplicative_decrease or something similar here. Or ad least spell out the name :slightly_smiling_face:

Reply:
That was my other thought :slightly_smiling_face: